routes/api.php

```diff
@@ -13,6 +11,10 @@
 |
 */
 
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
```

Comment:
Do you mean as a comment like the original.  Yeah I think that would be an improvement